<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskManagerSettingsDialog</class>
 <widget class="QDialog" name="TaskManagerSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Manager Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>530</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="showLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>351</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Showing tasks from:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showCurrentDesktopOnly">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Current desktop only</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="showCurrentScreenOnly">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>421</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Current screen only</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="groupTasksByApplication">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Group tasks by application</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TaskManagerSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TaskManagerSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
